src/go/rdctl/cmd/start.go

```diff
@@ -45,11 +45,13 @@ If it's running, behaves the same as 'rdctl set ...'.
 }
 
 var applicationPath string
+var noModalDialogs bool
 
 func init() {
 	rootCmd.AddCommand(startCmd)
 	options.UpdateCommonStartAndSetCommands(startCmd)
 	startCmd.Flags().StringVarP(&applicationPath, "path", "p", "", "Path to main executable.")
+	startCmd.Flags().BoolVarP(&noModalDialogs, "no-modal-dialogs", "", false, "Avoid displaying dialog boxes.")
```

Comment:
The description is normally lowercase; I guess you copied the format from `--path`, which was the sole exception, and should be modified as well:

```suggestion
	startCmd.Flags().StringVarP(&applicationPath, "path", "p", "", "path to main executable.")
	startCmd.Flags().BoolVarP(&noModalDialogs, "no-modal-dialogs", "", false, "avoid displaying dialog boxes.")
```

I think it would look even better if "Allowed values" would also be lowercase:

```
scripts/generateCliCode.ts
236:      usageParts.push(`(Allowed values: [${ rawEnums.join(', ' ) }])`);
```